app.js

```diff
@@ -1,8 +1,19 @@
-import { getDogs } from './fetch-utils.js';
-import { renderDogCard } from './render-utils.js';
+/* eslint-disable quotes */
+/* eslint-disable indent */
+import { getDogs } from "./fetch-utils.js";
+import { renderDogCard } from "./render-utils.js";
 
-const dogListContainer = document.getElementById('dog-list-container');
+const dogListContainer = document.getElementById("dog-list-container");
 
 // on load
-// fetch all dogs
```

Comment:
Great work here validating you have successfully hit your backend/supabase and by awaiting the results. Which you then render the dog and append the dogs/li to your container